src/Components/Steps/Expenses/Expenses.tsx

```diff
@@ -0,0 +1,323 @@
+          <HelpButton
+            helpText="Add up expenses for everyone who lives in your home. This includes costs like child care, child support, rent, medical expenses, heating bills, and more. We will ask only about expenses that may affect benefits. We will not ask about expenses such as food since grocery bills do not affect benefits."
+            helpId="questions.hasExpenses-description"
+          />
+        </div>
+      </QuestionQuestion>
+      <QuestionDescription>
+        <FormattedMessage
+          id="questions.hasExpenses-description-additional"
+          defaultMessage="Expenses can affect benefits! We can be more accurate if you tell us key expenses like your rent or mortgage, utilities, and child care."
+        />
+      </QuestionDescription>
+      <form onSubmit={handleSubmit(formSubmitHandler)}>
+        <Controller
+          name="hasExpenses"
+          control={control}
+          rules={{ required: true }}
+          render={({ field }) => (
+            <RadioGroup {...field} aria-labelledby={translatedAriaLabel} className="expense-radiogroup-margin-bottom">
+              <FormControlLabel
+                value={true}
+                control={<Radio />}
+                label={<FormattedMessage id="radiofield.label-yes" defaultMessage="Yes" />}
+              />
+              <FormControlLabel
+                value={false}
+                control={<Radio />}
+                label={<FormattedMessage id="radiofield.label-no" defaultMessage="No" />}
+              />
+            </RadioGroup>
+          )}
+        />
+        {fields.map((field, index) => {
+          const selectedExpenseSource = watch('expenses')[index].expenseSourceName;
+          return (
+            <Box className="expense-section-container" key={field.id}>
+              {index !== 0 && (
+                <div className="delete-button-container">
+                  <CloseButton handleClose={() => remove(index)} />
+                </div>
+              )}
+              <Stack className={index % 2 === 0 ? 'expense-section' : ''}>
+                <QuestionQuestion>
+                  {index === 0 ? (
+                    <div className="first-expense-q-padding">
+                      <FormattedMessage
+                        id="questions.hasExpenses-a"
+                        defaultMessage="What type of expense has your household had most recently?"
+                      />
+                    </div>
+                  ) : (
+                    <FormattedMessage
+                      id="expenseBlock.createExpenseBlockQuestions-questionLabel"
+                      defaultMessage="If you have another expense, select it below."
+                    />
+                  )}
+                </QuestionQuestion>
+              </Stack>
+              <FormControl
+                sx={{ m: 1, minWidth: '13.125rem', maxWidth: '100%' }}
+                error={!!errors.expenses?.[index]?.expenseSourceName}
```

Comment:
I think that this is a nicer way to write this. What do you think?
```suggestion
                error={errors.expenses?.[index]?.expenseSourceName !== undefined}
```

Reply:
I think it's fine as is.